src/pear/config.vsh.yaml

```
@@ -0,0 +1,145 @@
+functionality:
+  name: pear
+  description: "PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger"
+  info:
+    keywords: [ "pair-end", "read", "merge" ]
+    homepage: https://cme.h-its.org/exelixis/web/software/pear
+    documentatoin: https://cme.h-its.org/exelixis/web/software/pear/doc.html
+    reference: doi:10.1093/bioinformatics/btt593
+    licence: "['Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported']"
+  requirements:
+    cpus: 1
+    memory: 4GB
+    commands: [ pear , gzip ]
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: --forward_fastq
+          alternatives: -f
+          type: file
+          description: Forward paired-end FASTQ file
+          required: true
+          example: "forward.fastq"
+        - name: --reverse_fastq
+          alternatives: -r
+          type: file
+          description: Reverse paired-end FASTQ file
+          required: true
+          example: "reverse.fastq"
+    - name: Outputs
+      arguments:
+        - name: --output
+          alternatives: -o
+          type: string
+          description: The prefix of the output file(s). The output files will be named as <output>.assembled.fastq and <output>.unassembled.forward.fastq, <output>.unassembled.reverse.fastq and <output>.discarded.fastq.
+          required: true
+          example: "output.fastq"
+    - name: Arguments
+      arguments:
+        - name: --p_value
+          alternatives: -p
+          type: double
+          description: |
+            Specify a p-value for the statistical test. If the computed p-value of a possible assembly exceeds the specified p-value then  paired-end  read  will not be assembled. Valid options are: 0.0001, 0.001, 0.01, 0.05 and 1.0. Setting 1.0 disablesthe test.
+          default: 0.01
```

I'm inclined to set the default arguments as specified in the documentation as examples, in case the default used by the tool is not actually as what is written in the documentation.

```suggestion
          example: 0.01
```

Make these changes for all the other arguments as well.